src/redux/configureStore.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable*/
```

Comment:
- [ ] I will strongly advise you don't disable your linters because linters are to ensure that you follow best practices while writing code and also helps to diagnose and fix technical issues in your code. Kindly fix all your linter errors instead of disabling them 👍🏽